### PR TITLE
Improve documentation and examples

### PR DESCRIPTION
**Pull request by yadaiio**
Builds on top of #37 #20 and #14.

References: https://github.com/clue/reactphp-tar/pull/20, https://github.com/clue/reactphp-redis/pull/117 and https://github.com/clue/reactphp-sqlite/pull/43.

### CODE REVIEW

File: examples/random-colors.php [hunk 1]
```diff
@@ -23,10 +19,10 @@
 }
 
 // process control codes from STDIN
-$stdin = new ReadableResourceStream(STDIN);
-$parser = new ControlCodeParser($stdin);
+$stdin = new eact\Stream\ReadableResourceStream(STDIN);
```
Comment (clue) [inline examples/random-colors.php]:
```suggestion
$stdin = new React\Stream\ReadableResourceStream(STDIN);
```

File: README.md [hunk 2]
```diff
@@ -78,6 +82,10 @@ are supported as defined in [ISO/IEC 2022](https://en.wikipedia.org/wiki/ISO/IEC
 Each code sequence gets emitted with a dedicated event with its raw byte sequence:
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
```
Comment (clue) [inline README.md]: I don't think this adds a lot of value here given this is an incomplete snippet anyway, WDYT?

File: README.md [hunk 3]
```diff
@@ -36,6 +36,10 @@ It wraps a given `ReadableStreamInterface` and exposes its plain data through
 the same interface.
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
 $stdin = new React\Stream\ReadableResourceStream(STDIN);
 
 $stream = new ControlCodeParser($stdin);
```
Comment (clue) [inline README.md]:
```suggestion
$stream = new Clue\React\Term\ControlCodeParser($stdin);
```

File: examples/random-colors.php (hunk 1 above)
Comment (yadaiio) [reply examples/random-colors.php]: Oops, thank you for noticing this. Should be fixed now. 🎉

File: README.md (hunk 2 above)
Comment (yadaiio) [reply README.md]: Yes, thank you. 👍

File: README.md (hunk 3 above)
Comment (yadaiio) [reply README.md]: Should be added now, thank you very much @clue 🥳

**Review by clue — APPROVED** [review]
@yadaiio Thanks for the update, changes LGTM! :shipit:

**Pull request merged**